bricksrc/equipment.py

```diff
@@ -4,8 +4,8 @@
 Set up subclasses of the equipment superclass
 """
 equipment_subclasses = {
-    "HVAC": {"tags": [TAG.HVAC]},
-    "Weather_Station": {"tags": [TAG.Weather, TAG.Station]},
+    "HVAC": {"tags": [TAG.HVAC, TAG.Equipment]},
```

Comment:
We now have System category. Do we want to add `HVAC_System` there in the near future? In that case, I would like to make it more explicit as `HVAC_Equipment`. (It's a breaking change, so maybe we need to collect it for v2.)

Reply:
We don't need to remove "HVAC", but we should definitely add "HVAC Equipment" and clarify it as different from "HVAC System"

Reply:
It's quite confusing to have all of HVAC, HVAC_Equipment, and HVAC_System.

Reply:
Should we just move directly to HVAC Equip + HVAC System and remove HVAC in this release?

Reply:
I think you can get away without HVAC and without HVAC_Equipment and just have everything under Equipment. HVAC_System still makes sense to me